backend/entities/office_hours/oh_event_entity.py

```diff
@@ -28,12 +27,12 @@ class OfficeHoursEventEntity(EntityBase):
     type: Mapped[OfficeHoursType] = mapped_column(
         SQLAlchemyEnum(OfficeHoursType), nullable=False
     )
-    # Name of event
-    title: Mapped[str] = mapped_column(String, nullable=False)
```

Comment:
we realized we don't actually need a title for events, since we plan on using the event type as the label